bonusy/inspirace/assets/kolotoc.py

```diff
@@ -0,0 +1,49 @@
+import sys
+import random
+import re
```

Comment:
```suggestion
import random
import re
import sys
```

Reply:
To se ma spravne radit podle abecedy?

Reply:
Je to lepší. Existuje třeba modul isort, který řadí importy automaticky.